docs/quick-start/nginx-ingress/README.md

```diff
@@ -0,0 +1,131 @@
+
+Create the RBAC resources.
+
+```bash
+$ kubectl create serviceaccount tiller --namespace=kube-system
+$ kubectl create clusterrolebinding tiller-admin --serviceaccount=kube-system:tiller --clusterrole=cluster-admin
+```
+
+Now install the Tiller component.
+
+```bash
+$ helm init --service-account=tiller
+```
+
+*If you'd like to read more, full documentation is available on [installing Tiller](https://github.com/kubernetes/helm/blob/master/docs/install.md) and [RBAC configuration](https://github.com/kubernetes/helm/blob/master/docs/rbac.md).*
+
+
+## Step 1 - Deploy the NGINX Ingress Controller
+### Create a namespace
+```bash
+$ kubectl apply -f nginx/00-namespace.yaml
+```
+
+### Create a default backend
+Used for all requests that don't match any Ingress resource
+
+```bash
+$ kubectl apply -f nginx/default-deployment.yaml
+$ kubectl apply -f nginx/default-service.yaml
+```
+
+### Create the NGINX ingress controller
+```bash
+$ kubectl apply -f nginx/rbac.yaml
+$ kubectl apply -f nginx/configmap.yaml
+$ kubectl apply -f nginx/service.yaml
+$ kubectl apply -f nginx/deployment.yaml
+```
+
+The `nginx` ingress controller service is the only one which is made available outside the cluster, as it handles all incoming traffic. It does this via a load balancer in your cloud provider. A few minutes after you've created the `nginx` Service, get its public IP or domain name via `kubectl`.
+
+```bash
+$ kubectl describe svc nginx --namespace nginx-ingress
+[...]
+LoadBalancer Ingress:   1.2.3.4
+[...]
+```
+
+This is the IP to which all incoming traffic should be routed, so add it to a DNS zone you control, e.g. as `echo.example.com`. AWS will give a domain name instead; in this case CNAME your name to that name.
+
+
+## Step 2 - Deploy an Example Service (Echoserver)
+```bash
+$ kubectl apply -f echoserver/00-namespace.yaml
+$ kubectl apply -f echoserver/deployment.yaml
+$ kubectl apply -f echoserver/service.yaml
+```
+
+We will first create an Ingress resource that listens on plain HTTP.
+
+```bash
+$ kubectl apply -f echoserver/ingress-notls.yaml
+```
+
+Now make sure the echo service is reachable at the domain name you added above, e.g. http://echo.example.com.
+
+
+## Step 3 - Deploy Cert Manager
+```bash
+$ helm install --name cert-manager --namespace kube-system stable/cert-manager
+```
+
+## Step 4 - Configure Letsentrypt Issuer
+
+With cert manager running, we must tell it about a service it can use to issue certificates for us. In this case we will configure letsencrypt. NB: the following code sets up letsencrypt "production". This will issue valid certificates but has a strict rate limit. If you are having difficulty and failing to get certificates, [switch this for letsencrypt's "staging" issuer](https://github.com/jetstack/cert-manager/blob/master/docs/user-guides/acme-http-validation.md) while you debug, otherwise you will hit the rolling 7 day rate limit.
```

Comment:
Also let's use the staging issuer first, and once they've issued a certificate and verified it's issued by the Letsencrypt staging CA, we can instruct them how to switch over to prod (update the Issuer resource, and delete the TLS keypair secret that's signed by LE staging)